potodo/cache.py

```diff
@@ -14,6 +14,9 @@ def get_cache_file_content(
     cache_args: Any,
     path: str = ".potodo/cache.pickle",
 ) -> Dict[Path, PoFileStats]:
+    """
+    Get cache file content
```

Comment:
Gets the cache content from `path`, returning an empty dict if the args in the cache are different from `cache_args`